catalog/ui/catalog-ui-search/src/main/webapp/index.html

```diff
@@ -24,6 +24,8 @@
     <meta name="description" content="">
     <meta name="author" content="">
 
+    <link id="favicon" rel="icon" type="image/png" href="data:image/png;base64,">
```

Comment:
just curious, is the comma needed in the href?

Reply:
I believe so. From https://en.wikipedia.org/wiki/Data_URI_scheme: The minimal data URI is `data:,`, consisting of the scheme, no media-type, and zero-length data. So `image/png` and `;base64` are optional but the comma is necessary.